src/components/interaction-designer/clipboard/block/MessageBlock.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <div class="card" :class="{'disabled-block': !isFocused}">
```

Comment:
This is a common class(defined in ClipboardRoot) which adds a grey background to all the previous blocks (indicating that it cannot be edited directly). I guess changing the name would make more sense to something like grey-background. Let me know your thoughts on this

Reply:
I have a refactor change, due to which the card will move into ClipboardRoot. Hence, this change will not be necessary.

Reply:
@poojahpatil90 , I'm a bit confused here, we can chat tmr about this on slack to quickly finalise my review on this PR
Anyway, my concern here was about inserting a class name like `message-block` for the component name is `MessageBlock.vue`, eg: instead of `class="card"`, we have `class="card message-block"` (and so on for other .vue component)